mock-abis/src/main/java/io/mosip/proxy/abis/Listener.java

```diff
@@ -99,13 +105,17 @@ public boolean consumeLogic(javax.jms.Message message, String abismiddlewareaddr
 		boolean isrequestAddedtoQueue = false;
 		String messageData = null;
 		logger.info("Received message " + message);
-
-		if (!(message instanceof TextMessage)) {
-			return false;
-		}
 		try {
-			TextMessage textMessage = (TextMessage) message;
-			messageData = textMessage.getText();
+			if (message instanceof TextMessage || message instanceof ActiveMQTextMessage) {
+				textType = 1;
+				TextMessage textMessage = (TextMessage) message;
+				messageData = textMessage.getText();
+			} else if (message instanceof ActiveMQBytesMessage) {
+				textType = 2;
+				messageData = new String(((ActiveMQBytesMessage) message).getContent().data);
+			} else
+				return false;
```

Comment:
will be better if u put logger before returning false

Reply:
ok